### PR TITLE
remove turbolinks, close #537

### LINKED ISSUE #537: remove turbolinks?
I'd like to remove turbolinks. 
It makes dealing with JavaScript load events harder than it should be (see the maps on group#show).
Any opinions?

### PR DESCRIPTION
**Pull request by lislis**

### CODE REVIEW

File: app/views/shared/_grouped_people.haml [hunk 1]
```diff
@@ -15,3 +15,4 @@
               It's you
             - if location_for?(person)
               %p #{person.city} #{person.country_name}
+g
```
Comment (til) [inline app/views/shared/_grouped_people.haml]: g? 😄

**Comment by sareg0** [comment]
hehe apart from Til's comment about the phantom 'g', this PR LGTM!!

**Comment by lislis** [comment]
Hihi, mystery g has been removed!

**Comment by til** [comment]
Then I approve 😄

**Comment by lislis** [comment]
high five

**Pull request merged**